packages/aws-cdk-lib/aws-stepfunctions-tasks/README.md

```diff
@@ -386,6 +386,42 @@ const task = new tasks.CodeBuildStartBuild(this, 'Task', {
 });
 ```
 
+### StartBuildBatch
+
+[StartBuildBatch](https://docs.aws.amazon.com/codebuild/latest/APIReference/API_StartBuildBatch.html) starts a batch CodeBuild for a project by project name.
+It is necessary to enable the batch build feature in the CodeBuild project.
+
+```ts
+import * as codebuild from 'aws-cdk-lib/aws-codebuild';
+
+const project = new codebuild.Project(this, 'Project', {
+  projectName: 'MyTestProject',
+  buildSpec: codebuild.BuildSpec.fromObjectToYaml({
+    version: 0.2,
+    batch: {
+      'build-list': [
+        {
+          identifier: 'id',
+          buildspec: 'version: 0.2\nphases:\n  build:\n    commands:\n      - echo "Hello, from small!"',
+        },
+      ],
+    },
+  }),
+});
+project.enableBatchBuilds();
```

Comment:
Can you please add a note below explaining the scenarios when `enableBatchBuilds()` returns `undefined`?
Eg:
```
**Note**: Batch builds are not supported for ... 

You can verify if batch builds are enabled by checking `enableBatchBuilds()` returned value:

```ts
const enabled = project.enableBatchBuilds();
if (enabled === undefined) {
  throw new Error('Batch build were not enabled');
}
```

Reply:
@lpizzinidev 

I have added only Notes.

The `enableBatchBuilds()` returns `undefined` exclusively for imported projects, and this return value persists regardless of the batch build's actual status (enabled or not) within those projects. Consequently, users working with imported projects must manually ensure that batch builds are enabled, as relying on the return value of `enableBatchBuilds()` is not informative.

I believe it suffices to include this as a note without detailing the process of verifying the return value of `enableBatchBuilds()`.
What do you think about that?
I'm sorry for my initial README desciption confused you...